### PR TITLE
feat(deps): updating production dependencies

### PR DESCRIPTION
**Pull request by vladikoff**
Doing this early in the train so we can see if anything broke.

So far it looked okay..

### CODE REVIEW

**Comment by vladikoff** [comment]
Seems like some breakage in SASS 
image

**Comment by vladikoff** [comment]
Reverted grunt-sass update for now

File: grunttasks/selectconfig.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports = function (grunt) {
     console.log('Using configuration files', process.env.CONFIG_FILES);
 
     // `server` is a shortcut to the server configuration
-    var serverConfig = require('../server/lib/configuration').root();
+    var serverConfig = require('../server/lib/configuration').getProperties();
```
Comment (shane-tomlinson) [inline grunttasks/selectconfig.js]: I'm *amazed* this is the only line that needs to be updated with all of the major release bumps.

File: package.json [hunk 2]
```diff
@@ -23,50 +23,50 @@
...
     "connect-cachify": "0.0.17",
     "connect-fonts-clearsans": "0.0.1",
     "connect-fonts-firasans": "0.0.4",
-    "consolidate": "0.10.0",
-    "convict": "0.8.0",
-    "cookie-parser": "1.3.4",
-    "cors": "2.6.0",
-    "express": "4.12.3",
+    "consolidate": "0.13.1",
+    "convict": "0.8.2",
```
Comment (pdehaan) [inline package.json]: Not sure if we try and merge https://github.com/mozilla/node-convict/pull/85 PR and then pin it to `1.0.0`.
/cc @zaach

File: grunttasks/selectconfig.js (hunk 1 above)
Comment (pdehaan) [inline grunttasks/selectconfig.js]: +1, Great job @vladikoff!
:star2:

File: package.json (hunk 2 above)
Comment (vladikoff) [inline package.json]: We can do that later

**Comment by vladikoff** [comment]
local tests pass too against a prod build, merging early so we can see if there are issues

**Comment by shane-tomlinson** [comment]
I was just going to r+ this for you.

**Comment by pdehaan** [comment]
@shane-tomlinson If you're looking for an r+ fix, feel free to go merge happy on https://github.com/shane-tomlinson/fxa-crypto-utils/pulls (I have no powers there &mdash; unrelated https://github.com/shane-tomlinson/fxa-crypto-utils/issues/9)

**Pull request merged**